lib/manager/kustomize/extract.ts

```diff
@@ -118,7 +118,7 @@ export function parseKustomize(content: string): Kustomize | null {
     return null;
   }
 
-  if (pkg.kind !== 'Kustomization') {
+  if (!['Kustomization', 'Component'].includes(pkg.kind)) {
```

Comment:
I think it would be good to set `depType` based on the pkg.kind

Reply:
I am not sure it make sense as `kind` does not relate to the dependencies themselves.
Although, It could make sense to use the field where they are found as the `depType` (e.g. `base`, `component`, `image`, `resource`).
What would you like to be done from here?

Reply:
I think that depType should be set to Kustomization or Component now that there are these two types

Reply:
Done, but I do not see how this semantic would be useful